Addressed review from @sindresorhus (#2203)
wemake-services/wemake-python-styleguide: README.md

       alt="invocation results">
</p>

> ### Example

Reviewer: Let's remove this section for now:
1. I will make screenshots with the same style as "Running"
2. I will have to think about some cool examples to showcase

Author: I have removed this section and I will push the PR again.  
Do check and let me know if this is fine.

Here's some poorly written code:
<p align="center">
  <img src="/docs/_static/bad_code_example.PNG"
       alt="Bad code example">
</p>

Running the linter using the command,
```bash
flake8 test.py
```
we get the following errors and warnings:
<p align="center">
  <img src="/docs/_static/code_issues_example.PNG"
       alt="Code issues">
</p>

This is how the code looks after rectifying all the errors/warnings:
<p align="center">
  <img src="/docs/_static/good_code_example.PNG"
       alt="Good code example">
</p>

The updated code is pretty readable and has a consistent structure that is lacking in the bad code. The linter also suggests to simplify the logic of the code which makes it less redundant.

See ["Usage" section](https://wemake-python-stylegui.de/en/latest/pages/usage/setup.html)
in the docs for more examples and integrations.

We also support [GitHub Actions](https://wemake-python-stylegui.de/en/latest/pages/usage/integrations/github-actions.html) as first class-citizens.
[Try it out](https://github.com/marketplace/actions/wemake-python-styleguide)!

> ## Strict is the new cool

Reviewer: Amazing work! You nailed this part 🚀

Author: Thanks!

Strict linting offers the following benefits to developers and companies:
1. Ensures consistency - no matter who works on it, the end product will always be the same dependable code

> 2. Helps avoid potential bugs - strict rules make sure that you don't contradict/make a fool of yourself in the future

Reviewer: ```suggestion
2. Helps avoid potential bugs - strict rules make sure that you don't make common mistakes
```

3. Efficient code reviews - each piece of code has a similar familiar style and syntax. If it passes all the checks, there's little left to review!
4. Less code revisions - strict linting ensures that you don't have to re-write the codebase again and again

> 5. Reduce code redundancy - Sometimes we write complex code as we are thinking in a certain way about a problem. The linter offers suggestions that can help simplify the code and eliminate redundant statements. 

Reviewer: ```suggestion
5. Reduce code redundancy - Sometimes we write complex code as we are thinking in a certain way about a problem. The linter offers suggestions that can help simplify the code and eliminate redundant statements
```
